charts/workflow/requirements.yaml

```diff
@@ -45,6 +45,7 @@ dependencies:
     version: <registry-token-refresher-tag>
     repository: https://charts.deis.com/registry-token-refresher
   - name: router
+    condition: !global.experimental_native_ingress
```

Comment:
But if it works... this is _exactly_ what I had in ming for how the router chart, in its entirety, is conditionally included or excluded. I ❤️ this.